protobuf/scalapb/scalapb.proto

```diff
@@ -177,6 +177,9 @@ message ScalaPbOptions {
   // If true, getters will be generated.
   optional bool getters = 27 [default=true];
 
+  // If true, complex fields would be lazy
+  optional bool lazy_fields = 100;
```

Comment:
can we remove this option and leave only the field-level option for now? Through field transformations feature users can specify rules that turn on `lazy` with more precision (only for repeated fields, only for certain types, etc).